src/constants.ts

```diff
@@ -9,8 +9,12 @@ export const DEFAULT_POLL_CONFIG_REQUEST_RETRIES = 7;
 export const BASE_URL = 'https://fscdn.eppo.cloud/api';
 export const UFC_ENDPOINT = '/flag-config/v1/config';
 export const BANDIT_ENDPOINT = '/flag-config/v1/bandits';
+export const EVENT_ENDPOINT = 'v0/i';
```

Comment:
should this have a leading slash?

Reply:
since we have slash-handling in the `ApiEndpoints`, it's not strictly necessary, but I like it for consistency